include/mbedtls/net_sockets.h

```diff
@@ -1,7 +1,22 @@
 /**
  * \file net_sockets.h
  *
- * \brief Network communication functions
+ * \brief   Network sockets abstraction layer to integrate Mbed TLS into a
+ *          BSD-style sockets API.
+ *
+ *          The network sockets module provides an example integration of the
+ *          Mbed TLS library into a BSD sockets implementation. The module is
```

Comment:
Although I am not a native English speaker, I agree with @irwir 's comment on using "both" and "also".
I would remove the word "both" and replace "and also" with "as well as"